test/verify/storagelib.py

```diff
@@ -463,9 +437,15 @@ def configuration_field(self, dev, tab, field):
     def wait_in_configuration(self, dev, tab, field, text):
```

Comment:
All of these `wait_` variants are now unused and can be removed.

Reply:
Ok, done!